transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -417,17 +394,18 @@ final ChannelHandler handler() {
     }
 
     final Map<ChannelOption<?>, Object> options() {
-        return copiedMap(options);
+        return Collections.unmodifiableMap(options);
```

Comment:
I think this should copy to preserve the existing semantics https://github.com/netty/netty/blob/695aa0959dcc7fb97f6d0e0fa26a45bed32f730a/transport/src/main/java/io/netty/bootstrap/AbstractBootstrapConfig.java#L63